gst/parser/gstttmlparse.c

```diff
@@ -1607,8 +1607,9 @@ handle_buffer (GstTtmlParse * self, GstBuffer * buf)
   if (g_strcmp0 (self->subtitle_codec, "EBUTT") == 0) {
     GList *subtitle;
     GTimer *timer = g_timer_new ();
-    CParser *ttml_parser = parse_ttml (self->textbuf->str, self->forced_only);
-    if (ttml_parser == NULL) {
+
+    CParser* subs_parser = create_subs_parser();
```

Comment:
I guess we should check for `NULL` here, no?

Reply:
Added that